### PR TITLE
fix: remove reference to this repository being outside of the cypress monorepo

### PR DESCRIPTION
**Pull request by JessicaSachs**
Need to publish to the latest tag. Also, remove the old notice because this is not on the bahmutov namespace anymore

### CODE REVIEW

File: npm/vue/.releaserc.js [hunk 1]
```diff
@@ -2,6 +2,6 @@ module.exports = {
   ...require('../../.releaserc.base'),
   branches: [
     '@cypress/vue@1.0.0',
-    { name: 'master', prerelease: 'alpha' },
+    { name: 'master', channel: 'latest', prerelease: 'alpha' },
```
Comment (elevatebart) [inline npm/vue/.releaserc.js:5]: Channel latest is the default one, next is the one you want!

```suggestion
    { name: 'master', channel: 'next', prerelease: 'alpha' },
```

**Review by elevatebart — APPROVED** [review]

**Review by elevatebart — CHANGES_REQUESTED** [review]
change it when you can.

File: npm/vue/.releaserc.js (hunk 1 above)
Comment (JessicaSachs) [reply npm/vue/.releaserc.js:5]: Hmm... I think I want latest?... There's no stable version.

Comment (JessicaSachs) [reply npm/vue/.releaserc.js:5]: Hmm... I think I want latest?... There's no stable version.

**Review by elevatebart — DISMISSED** [review]

**Review by jennifer-shehane — APPROVED** [review]

**Comment by jennifer-shehane** [comment]
@JessicaSachs Is this still relevant?

**Comment by jennifer-shehane** [comment]
Unfortunately we have to close this PR due to inactivity. Please open a new PR addressing the original issue and any requested changes.

**Pull request closed without merging**